vendor/bat-native-ledger/src/bat/ledger/internal/properties/publisher_settings_properties.h

```diff
@@ -0,0 +1,41 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVELEDGER_PUBLISHER_SETTINGS_PROPERTIES_H_
+#define BRAVELEDGER_PUBLISHER_SETTINGS_PROPERTIES_H_
+
+#include <stdint.h>
+#include <map>
+#include <string>
+#include <vector>
+
+#include "bat/ledger/internal/properties/report_balance_properties.h"
+
+namespace ledger {
+
+struct PublisherSettingsProperties {
+  PublisherSettingsProperties();
+  PublisherSettingsProperties(
+      const PublisherSettingsProperties& properties);
+  ~PublisherSettingsProperties();
+
+  bool operator==(
+      const PublisherSettingsProperties& rhs) const;
+
+  bool operator!=(
+      const PublisherSettingsProperties& rhs) const;
+
+  uint64_t minimum_page_time_before_logging_a_visit;
```

Comment:
could we have a shorter name? maybe `minimum_logging_time` or `min_visit_time` or something like this? I think `min` vs `minimum` would help as well

Reply:
The problem previously was code was hard to read to reconcile with UI. At a minimum, I think `min_page_time_before_logging_a_visit`

Reply:
Fixed as above